lib/actions/user.js

```diff
@@ -528,11 +605,11 @@ export function saveUserPlace (placeToSave, placeIndex, intl) {
 /**
  * Delete the place data at the specified index for the logged-in user.
  */
-export function deleteUserPlace (placeIndex, intl) {
+export function deleteUserPlace(placeIndex, intl) {
   return function (dispatch, getState) {
-    if (
-      !confirm(intl.formatMessage({ id: 'actions.user.confirmDeletePlace' }))
-    ) return
+    // eslint-disable-next-line no-restricted-globals
+    if (!confirm(intl.formatMessage({ id: 'actions.user.confirmDeletePlace' })))
```

Comment:
Same here, does `window.confirm` let us remove the linting exception?

Reply:
Absolutely does! Good shout

Done in 90efea159fb187b6d36a42607c00df74e0880d54